src/Tests/Helpers/ServiceBusMessageHelper.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace SFA.DAS.Funding.SystemAcceptanceTests.Helpers
+{
+    internal class ServiceBusMessageHelper
+    {
+        private readonly ScenarioContext _context;
+        private ApprenticeshipCreatedEvent _apprenticeshipCreatedEvent;
+        private EarningsGeneratedEvent _earnings;
+        private FundingPeriod _funding;
+
+        public ServiceBusMessageHelper(ScenarioContext context)
+        {
+            _context = context;
+        }
+
+        public void CreateAnApprenticeshipMessage(DateTime actualStartDate, DateTime plannedEndDate, decimal agreedPrice)
```

Comment:
This method name hides the fact it's overwriting whatever is in _apprenticeshipCreatedEvent. Maybe drop the "An" from this and the publish method?

Reply:
Updated